service/debugger/debugger.go

```diff
@@ -701,6 +716,13 @@ func (d *Debugger) AmendBreakpoint(amend *api.Breakpoint) error {
 	defer d.targetMutex.Unlock()
 
 	originals := d.findBreakpoint(amend.ID)
+	if originals == nil {
+		if _, ok := d.disabledBreakpoints[amend.ID]; ok {
+			d.disabledBreakpoints[amend.ID] = amend
+		}
+		return nil
```

Comment:
This needs to return an error like we did before (the `if originals == nil` below can never be reached).